README.md

```diff
@@ -28,31 +28,159 @@ flag is highly recommended for those new to [pip](https://pip.pypa.io).
 
-The API requires an account for use. [Signup here](https://www.planet.com/explorer/?signup).
+```python
+def sync_create_order(order_details):
+    return asyncio.run(create_order(order_details))
 
-This can be provided via the environment variable `PL_API_KEY` or the flag `-k` or `--api-key`.
+oid = sync_create_order(order_details)
+print(oid)
+```
 
-Using `planet init` your account credentials (login/password) can be used to obtain the api key.
+When using `asyncio.run` to develop synchronous code with the async library,
+keep in mind this excerpt from the
+[asyncio.run](https://docs.python.org/3/library/asyncio-task.html#asyncio.run)
+documentation:
+
+"*This function always creates a new event loop and closes it at the end. It
+should be used as a main entry point for asyncio programs, and should ideally
+only be called once.*"
+
+Do you have a use case where native synchronous support is essential? If so,
+please contribute to
+[Determine need for synchronous support](https://github.com/planetlabs/planet-client-python/issues/251)
+
+
+Why async? Because things get *really cool* when you want to work with multiple
+orders. Here's an example of submitting two orders, waiting for them to
+complete, and downloading them. The orders each clip a set of images to a
+specific area of interest (AOI), so they cannot be combined into one order.
+(hint: [Planet Explorer](https://www.planet.com/explorer/) was used to define
+the AOIs and get the image ids.)
+ 
+
+```python
+import asyncio
+import os
+
+import planet
+
+API_KEY = os.getenv('PL_API_KEY')
+
+iowa_aoi = {
+    "type": "Polygon",
+    "coordinates": [[
+        [-91.198465, 42.893071],
+        [-91.121931, 42.893071],
+        [-91.121931, 42.946205],
+        [-91.198465, 42.946205],
+        [-91.198465, 42.893071]]]
+}
+
+iowa_images = [
+    '20200925_161029_69_2223',
+    '20200925_161027_48_2223'
+]
+iowa_order = planet.OrderDetails(
+    'iowa_order',
+    [planet.Product(iowa_images, 'analytic', 'PSScene4Band')],
+    tools=[planet.Tool('clip', {'aoi': iowa_aoi})]
+)
+
+oregon_aoi = {
+    "type": "Polygon",
+    "coordinates": [[
+        [-117.558734, 45.229745],
+        [-117.452447, 45.229745],
+        [-117.452447, 45.301865],
+        [-117.558734, 45.301865],
+        [-117.558734, 45.229745]]]
+}
+
+oregon_images = [
+    '20200909_182525_1014',
+    '20200909_182524_1014'
+]
+oregon_order = planet.OrderDetails(
+    'oregon_order',
+    [planet.Product(oregon_images, 'analytic', 'PSScene4Band')],
+    tools=[planet.Tool('clip', {'aoi': oregon_aoi})]
+)
+
+
+async def create_and_download(order_detail, client):
+    oid = await client.create_order(order_detail)
+    print(oid)
+    state = await client.poll(oid, verbose=True)
+    print(state)
+    filenames = await client.download_order(oid, progress_bar=True)
+    print(f'downloaded {oid}, {len(filenames)} files downloaded.')
+
+
+async def main():
+    async with planet.Session(auth=(API_KEY, '')) as ps:
```

Comment:
I have the same concern here as above with regard to synchronization and using the abstract function call instead of the authentication tuple.

Reply:
the authentication work is captured in #248 - this will be changed there.